package.json

```diff
@@ -45,12 +45,8 @@
   },
   "homepage": "https://github.com/bcherny/json-schema-to-typescript#readme",
   "dependencies": {
-    "@types/cli-color": "^0.3.29",
     "@types/json-schema": "^7.0.3",
-    "@types/lodash": "^4.14.121",
-    "@types/minimist": "^1.2.0",
-    "@types/mz": "0.0.32",
-    "@types/node": "^11.10.4",
+    "@types/node": "*",
```

Comment:
Could we add a lower bound here, eg. `>=6.0.0` or something reasonable?

Reply:
Set to `>= 4.5.0`, same as `engines` property.